docs/plugins.rst

```diff
@@ -177,6 +177,27 @@ then be able to log in with their Google account when they click the log in page
 and select the option to log in with Google.
 
 
+Curation
+--------
+
+This plugin adds curation functionality to Girder, similar to that in MIDAS.
```

Comment:
suggested

This plugin adds curation functionality to Girder, allowing content to be assembled and approved prior to publication.  Admin users can activate curation for any folder, and users who are then granted permission can compose content under that folder.  The users can request publication of the content when it is ready, which admins may approve or reject.  The plugin provides a UI along with workflow management, notification, and permission support for these actions.